invenio/modules/search/enhancers/collection_filter.py

```diff
@@ -0,0 +1,96 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Invenio.
+# Copyright (C) 2015 CERN.
+#
+# Invenio is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# Invenio is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Invenio; if not, write to the Free Software Foundation, Inc.,
+# 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+
+from invenio.base.globals import cfg
+from invenio_query_parser.ast import (
+        AndOp, OrOp, NotOp, KeywordOp, Keyword, Value
+)
+
+def kw_formatter(val):
+    return KeywordOp(Keyword("collection"), Value(val))
```

Comment:
shouldn't it be `collection:"value"` hence `KeywordOp(Keyword("collection"), DoubleQuotedValue(val))`?